<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPlot3D</class>
 <widget class="QWidget" name="TabPlot3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Plot3D" name="plot3d" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Метод</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="cbRungeKutta3d">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Метод Рунге-Кутты 4-го порядка</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/icons/green.png</normaloff>:/icons/icons/green.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="cbModEulers3d">
        <property name="text">
         <string>Модифицированный метод Эйлера</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/icons/red.png</normaloff>:/icons/icons/red.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="cbEulers3d">
        <property name="toolTip">
         <string>Синий</string>
        </property>
        <property name="text">
         <string>Метод Эйлера</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/icons/blue.png</normaloff>:/icons/icons/blue.png</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="cbAdams3d">
        <property name="text">
         <string>Метод Адамса-Вашфорта 4-го порядка</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/icons/magenta.png</normaloff>:/icons/icons/magenta.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plot3D</class>
   <extends>QWidget</extends>
   <header>plot3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
